invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/EItemDetails/components/EItemMetadata/EItemMetadata.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Grid, Segment, Container, Header, Table } from 'semantic-ui-react';
+import { openRecordEditor } from '../../../../../routes/urls';
+import { eitem as eitemApi } from '../../../../../common/api';
+import { EditButton } from '../../../components/buttons';
+
+import './EItemMetadata.scss';
+
+export default class EItemMetadata extends Component {
+  _renderUrlLinks = urls =>
```

Comment:
Just a proposal here, personally I consider the underscores in class methods noise. It's not Python, and as you see from React i.e. calls it `render` not `_render`.

Reply:
I agree I just followed the same style as all the other ones. Another refactor maybe? Is it make sense to change it only here?